src/components/Home.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faShuffle } from '@fortawesome/free-solid-svg-icons';
+import '../styles/Home.scss';
+
+const getRandomInt = (max) => {
+    return Math.floor(Math.random() * Math.floor(max));
+};
+const getRandomQuote = (movies) => {
+    let line = '';
+    let randMovie = null;
+    let randQuote = null;
+    while (line.split(' ').length <= 4 || line[0] !== line[0].toUpperCase() || line[0].toLowerCase() === line[0].toUpperCase()) {
```

Comment:
I have a hard time understanding this `line[0].toLowerCase() === line[0].toUpperCase()`

you are comparing `a` to `A`, it will always be false or am I missing something?